crates/astria-sequencer-client/src/lib.rs

```diff
@@ -248,60 +197,54 @@ impl ErrorKind {
             rpc,
         })
     }
-
-    /// Convenience method to construct a `WrongQueryResponseKind` variant.
-    fn wrong_query_response_kind(
-        expected: &'static str,
-        received: sequencer::accounts::query::Response,
-    ) -> Self {
-        Self::WrongQueryResponseKind(WrongQueryResponseKind {
-            expected,
-            received,
-        })
-    }
 }
 
 /// Tendermint HTTP client which is used to interact with the Sequencer node.
 #[async_trait]
 pub trait SequencerClientExt: Client {
     /// Returns the balance of the given account at the given height.
     ///
-    /// If `height = None`, the latest height is used.
-    ///
     /// # Errors
     ///
     /// - If calling tendermint `abci_query` RPC fails.
-    /// - If the opaque bytes cannot be deserialized as a sequencer query response.
-    /// - If the deserialized sequencer query response is not a balance response.
-    async fn get_balance(
-        &self,
-        address: &Address,
-        height: Option<Height>,
-    ) -> Result<Balance, Error> {
-        use borsh::BorshDeserialize as _;
-        use sequencer::accounts::query::Response;
-        let height = height.map(Into::into);
+    /// - If the bytes contained in the abci query response cannot be read as an
+    ///   `astria.sequencer.v1alpha1.BalanceResponse`.
+    async fn get_balance(&self, address: [u8; 20], height: u32) -> Result<BalanceResponse, Error> {
+        use proto::Message as _;
+
+        let prefix = b"accounts/balance/";
```

Comment:
make this a const (or put it in `sequencer` query module?)

Reply:
I made the prefixes const. I guess we could expose them somewhere, but I think it's fine if they remain private for now.

Not sure which `sequencer` query module you mean? If `sequencer::accounts::query` then I think no, because it's the responsibility of `sequencer::service::info` to route the requests to the relevant helpers.